tck/src/main/java/org/eclipse/microprofile/rest/client/tck/ssl/SslContextTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.microprofile.rest.client.tck.ssl;
+
+import org.apache.http.ssl.SSLContextBuilder;
+import org.eclipse.microprofile.rest.client.RestClientBuilder;
+import org.eclipse.microprofile.rest.client.tck.interfaces.JsonPClient;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.testng.annotations.Test;
+
+import javax.net.ssl.SSLContext;
+import javax.ws.rs.ProcessingException;
+
+
+public class SslContextTest extends AbstractSslTest {
+
+    @Deployment
+    public static WebArchive createDeployment() {
+        WebArchive webArchive = ShrinkWrap.create(WebArchive.class)
```

Comment:
For the tests, can you add the test class's simple name as the archive names?
```suggestion
        WebArchive webArchive = ShrinkWrap.create(WebArchive.class, SslContextTest.class.getSimpleName() + ".war")
```